### PR TITLE
Check if the accessibility app is enabled

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes https://github.com/nextcloud/server/issues/19177

### CODE REVIEW

File: src/views/RichWorkspace.vue [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default {
 			loaded: false,
 			ready: false,
 			autofocus: false,
-			darkTheme: OCA.Accessibility.theme === 'dark',
+			darkTheme: typeof OCA.Accessibility !== 'undefined' && OCA.Accessibility.theme === 'dark',
```
Comment (skjnldsv) [inline src/views/RichWorkspace.vue]: Why not just  ?
```suggestion
			darkTheme: OCA && OCA.Accessibility && OCA.Accessibility.theme === 'dark',
```

Comment (juliushaertl) [reply src/views/RichWorkspace.vue]: Well that still throws an ReferenceError

Comment (skjnldsv) [reply src/views/RichWorkspace.vue]: OCA is not defined?

Comment (juliushaertl) [reply src/views/RichWorkspace.vue]: Ah never mind, to early :see_no_evil:

**Pull request merged**